pkg/ruler/ruler.go

```diff
@@ -518,6 +563,68 @@ func instanceOwnsRuleGroup(r ring.ReadRing, g *rulespb.RuleGroupDesc, disabledRu
 	return ownsRuleGroup, nil
 }
 
+func (r *Ruler) LivenessCheck(_ context.Context, request *LivenessCheckRequest) (*LivenessCheckResponse, error) {
+	level.Debug(r.logger).Log("msg", "liveness check request", "request received from", request.RulerAddress)
```

Comment:
Seems `RulerAddress` is only used here for debug logging purpose. Can we remove that?

Reply:
The whole log is a debug log line and the log is not useful if it does not have the ruler address. If someone wants to troubleshoot, having the ruler address that performed the liveness check would be useful I think.

Reply:
humm we cannot log it in the client side? wouldn't this have the same effect?

Reply:
Removed